cirrus/server/cirrus/sdk.py

```diff
@@ -2,14 +2,47 @@
 import logging
 from typing import Any, Dict, List
 
-from cirrus_sdk import CirrusClient, NimbusError  # type: ignore
+from cirrus_sdk import (  # type: ignore
+    CirrusClient,
+    EnrollmentStatusExtraDef,
+    MetricsHandler,
+    NimbusError,
+)
 
 logger = logging.getLogger(__name__)
 
 
+class CirrusMetricsHandler(MetricsHandler):
+    def __init__(self, metrics: Any, pings: Any):
```

Comment:
Sad type signature is sad 😭

Reply:
Yeah I was hoping there'd be a specific type for the metrics/pings but the return type of the Glean functions is listed as `Any`